include/ardpolyfills/Servo.h

```diff
@@ -0,0 +1,49 @@
+/*
+ *  Servo.h
+ *  Copyright 2020 ItJustWorksTM
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+#ifndef SMARTCAR_EMUL_SERVO_H
+#define SMARTCAR_EMUL_SERVO_H
+
+#include <cstdint>
+
+#define MIN_PULSE_WIDTH      1000
+#define MAX_PULSE_WIDTH      2000
+#define DEFAULT_PULSE_WIDTH  1500
+
+#define INVALID_SERVO         255
```

Comment:
Don't use macros; they have terrible semantics. Use anonymous enums instead

Reply:
I've used them [here](https://github.com/DIT112-V20/group-09/blob/65fd2c71efdf5f9412efc0e8c8068e6b7ae11122/include/components/MovableCamera.hxx#L35) and that got merged no problem, any reason why its different now?

Reply:
Because Urho3D keys are enums, and you cannot make an enum derived from another enum
Can't use global constexpr either to avoid being able to take the address of it